server/src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -2177,9 +2177,10 @@ public Response bind(
             }
             catch (AutobindDisabledForOwnerException e) {
                 if (owner.isUsingSimpleContentAccess()) {
-                    throw new BadRequestException(i18n.tr("Ignoring request to auto-attach. " +
+                    log.debug("Ignoring request to auto-attach. " +
                         "It is disabled for org \"{0}\" because of the content access mode setting."
-                        , owner.getKey()));
+                        , owner.getKey());
+                    return null;
```

Comment:
Instead of just returning null we should build a Response so that it is clear what we are trying to accomplish.

Reply:
Agreed. A response with status 200 set on it

Reply:
Done.